lib/newrelic_httprb/wrappers.rb

```diff
@@ -0,0 +1,51 @@
+module NewRelicHTTP
+  class HTTPResponse
+    attr_reader :response
+
+    def initialize(response)
+      @response = response
+    end
+
+    def [](key)
+      response.headers.each do |k,v|
+        if key.downcase == k.downcase
+          return v
+        end
+      end
+      nil
+    end
+
+    def to_hash
+      response.headers
+    end
+  end
```

Comment:
Is this a copy of `NewRelic::Agent::HTTPClients::HTTPClientResponse`? I don't think their license allows this usage. Alternatively, we can just do

```ruby
module NewRelicHTTP
  HTTPResponse = NewRelic::Agent::HTTPClients::HTTPClientResponse
```

...I think.